include/gl_depth_sim/interfaces/noise_model.h

```diff
@@ -0,0 +1,17 @@
+#ifndef NOISE_MODEL_H
+#define NOISE_MODEL_H
+
+#include <math.h>
+#include <vector>
+#include <iostream>
+#include <iomanip>
```

Comment:
I think `string`, `iostream`, `iomanip`, `map` are left over copy paste from an example. My gut tells me that only `vector`, `math`, and `random` are needed.